lib/redis-client.js

```diff
@@ -1,6 +1,12 @@
 'use strict'
 
 module.exports = RedisClient
+// todo: to move this into it's own file
+class SearchHelper {
```

Comment:
Actually, sorry for not being clear: I think almost the entire implementation of `RedisClient.getLinks()` should be extracted into its own class, because there are dependencies between the local variables and the defined functions. Something like:

```js
function SearchHelper(searchString) {
  this.links = []
  this.cursor = 0
  this.regExp = '/*' + (searchString ? searchString + '*' : '')
}
```
...and then adding each function, etc. Mind doing that? (You can use the new `class` style if you want; I'm just more familiar with the old syntax so I used it here.)

Reply:
Sure, will do that 👍

Reply:
Also, you can take that TODO comment out. Not sure that class will ever need its own module, since it's an implementation detail well-covered by the existing tests. (But being its own class, it's easy to retain that option and add class-specific tests in the future.)